fMRIVolume/GenericfMRIVolumeProcessingPipeline.sh

```diff
@@ -293,13 +340,29 @@ if [ ! -e "$fMRIFolder" ] ; then
   log_Msg "mkdir ${fMRIFolder}"
   mkdir "$fMRIFolder"
 fi
-cp "$fMRITimeSeries" "$fMRIFolder"/"$OrigTCSName".nii.gz
+${FSLDIR}/bin/imcp "$fMRIScout" "$fMRIFolder"/"$OrigScoutName"
+
+# --- Do slice time correction if indicated
+if [ $DoSliceTimeCorrection = "TRUE" ] ; then
+    log_Msg "Running slice timing correction"
+    SliceTimerCorrectionParameters=`echo ${SliceTimerCorrectionParameters} | sed 's/@/ /g'` 
+    ${FSLDIR}/bin/slicetimer -i "$fMRIFolder"/"$OrigTCSName" -o "$fMRIFolder"/"$OrigTCSName"_orig_stc -r ${TR} -v ${SliceTimerCorrectionParameters}
+    
+    if [ "${DistortionCorrection}" = "NONE"] ; then {
+      #  use FSL's fslreorient2std for reorienting the image to match the approximate orientation of the standard template images MNI152
+      #  this makes the single-band processing more robust to registration problems.
+      #  NOTE: Take into account that BOLD images might have a different spatial orientation!
```

Comment:
I'd like for the comment here to capture more of our discussion as to why this is only allowed if DistortionCorrection=NONE.  e.g., I propose the following as a replacment:
```
# Processing is more robust to registration problems if the fMRI is in the same orientation as the
# standard template (MNI152) images, which can be accomplished using FSL's `fslreorient2std`.
# HOWEVER, if you reorient, other parameters (such as UnwarpDir) need to be adjusted accordingly.
# Rather than deal with those complications here, we limit reorienting to DistortionCorrection=NONE condition.
```

Reply:
Done.